edx-platform/pearson-vue-theme/lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -16,9 +17,9 @@
   <div class="data-display-text" id="data-grade-config-text"></div>
   <br>
 
-  <p>${_("The CSV of anonymized student IDs download has been relocated to the Reports section below.")}</p>
-
- </div>
+  <p>${_("Click to download a CSV of anonymized student IDs:")}</p>
+  <p><input type="button" name="list-anon-ids" value="${_("Get Student Anonymized IDs CSV")}" data-csv="true" class="csv" data-endpoint="${ section_data['get_anon_ids_url'] }" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}" ></p>
```

Comment:
Is this part of the Olive template? @sergivalero20

Reply:
@Squirrel18  Yes, It is.